manifests/a/Avogadro/Avogadro/1.2.0/Avogadro.Avogadro.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Avogadro.Avogadro
+PackageVersion: 1.2.0
+PackageLocale: en-US
+Publisher: Avogadro

```

```suggestion
Publisher: Humanity
```
`Publisher` must match Control Panel.